Makefile

```diff
@@ -28,10 +28,13 @@ CODESPELL_ARGS = -D - -D .github/codespell-dictionary.txt \
 
 all: lint-fix lint test gen web  ## Lint, build, and test everything
 
```

Comment:
Automatically derive the width of the commands that will be rendered.  Also fixes a bug where a multi-file Makefile will break the scanner.